### PR TITLE
Remove Python client and lib; update information on firmware

### PR DESCRIPTION
**Pull request by whoenig**
This follows through with the discussion at
https://github.com/USC-ACTLab/crazyswarm/pull/447, namely:

* Remove the (out-of-date) submodules of client and lib
* Update the documentation to advice users to use cfclient for firmware
  updates. This is required, because the most recent update procedure
  also updates the deck firmware (e.g., LightHouse deck).
* Clarify in the documentation, that the Crazyradio firmware has to be
  updated, even for new radios (they ship with an old firmware).
* Remove the prebuilt firmware for STM32 and NRF51 firmwares. Instead,
  specify the last tested official release in the documentation.

### CODE REVIEW

**Comment by whoenig** [comment]
@The-SS, perhaps you can give your input as well, as this is related to your original PR https://github.com/USC-ACTLab/crazyswarm/pull/447 (which we would close once this one is merged).

**Comment by jpreiss** [comment]
CI failure is due to #476

**Comment by jpreiss** [comment]
Does the new flashing procedure mean we cannot flash a batch of CFs from `chooser.py` anymore? That would be bad.

**Comment by The-SS** [comment]
Sorry for the delayed response. 
I think the changes make sense, but would like to note the following as well:
- Regarding the client and lib submodules, I would like to bring [issue 448](https://github.com/USC-ACTLab/crazyswarm/issues/448) up again since it might be relevant. Summary: The newer Bitcraze packages require a specific version of numpy that causes some crazyswarm features to fail (simulator specifically).
- It would be nice to still be able to use `chooser.py`. Either way, the documentation should at least mention whether or not it will work.
- It would be good to link the following in the documentation:
   - [To flash STM32](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/building-and-flashing/build/)
   - [To flash NRF51](https://github.com/bitcraze/crazyflie2-nrf-firmware/blob/master/docs/build/build.md)

**Comment by whoenig** [comment]
This is almost only a doc-change - chooser.py will keep working and the default paths on where it looks is unaffected by the file deletions. I added a new commit to reflect that in the docs.

@The-SS: My understanding is that #448 should be solved now from the bitcraze side - lets continue the discussion there. I didn't end up putting the detailed links on how to flash the firmware yet, since I wasn't sure where it should be without overloading the users? For cfclient users it is easiest to just flash the latest release directly from the GUI. For people who compile their own firmware, they should use chooser.py (for a swarm) or know the manual process already (while debugging/working on a single CF).

File: docs/configuration.rst [hunk 1]
```diff
@@ -25,26 +25,20 @@ Note: Crazyflies must be rebooted after any change of the channel/address for th
...
+1. Upgrade the firmwares of your Crazyflies with the latest official release using `cfclient`. This will update STM32, NRF51, and attached decks (e.g., LightHouse deck). We tested version 2021.6.
 
-    #. Plug in a battery
-    #. Turn your Crazyflie off by pressing the on/off button
-    #. Set your Crazyflie into bootloader mode by holding the on/off button for 3 seconds (The blue M2 and M3 LEDs start to blink)
-    #. ``rosrun crazyflie_tools flash --target nrf51 --filename prebuilt/cf2_nrf.bin``
-    #. Turn your Crazyflie off by pressing the on/off button
-    #. Set your Crazyflie into bootloader mode by holding the on/off button for 3 seconds (The blue M2 and M3 LEDs start to blink)
-    #. ``rosrun crazyflie_tools flash --target stm32 --filename prebuilt/cf2.bin``
+    Note: If you use a custom-built firmware, you can also use `chooser.py`, see below.
```
Comment (The-SS) [inline docs/configuration.rst:39]: Note: To use CLI check out the official Bitcraze documentation:  [STM32](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/building-and-flashing/build/), [NRF51](https://github.com/bitcraze/crazyflie2-nrf-firmware/blob/master/docs/build/build.md)

**Comment by The-SS** [comment]
@whoenig,
- I closed issue #448 
- Thanks for adding the note about using chooser.py
- I added a comment to the "file changes" with my suggestion on adding the STM/NRF CLI documentation links. It would just be another note after the chooser.py note. I agree that it is not necessary, but I think it would be nice to have there.

**Review by jpreiss — APPROVED** [review]

**Pull request merged**